exasol/ds/sandbox/lib/pretty_print.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime, timedelta
+
+
+def elapsed(start: datetime, round_to_seconds=True) -> str:
+    d = datetime.now() - start
+    if round_to_seconds:
+        d = d - timedelta(microseconds=d.microseconds)
+    return str(d)
+
+
+def size(num: int, factor: float = 1024, unit: str = "B") -> str:
```

Comment:
There is package called humanfriendly which might help here. We use it already in other packages